package.json

```diff
@@ -52,7 +52,7 @@
     "ci:test-frontend": "yarn run test:ci && yarn grafana-toolkit node-version-check",
     "i18n:extract": "lingui extract",
     "i18n:compile": "lingui compile",
-    "postinstall": "husky install",
+    "postinstall": "husky install && node ./scripts/copy-unicons.js",
```

Comment:
This will ensure the icons are available to build either packages or grafana itself.